www/include/core/header/header.iframe.php

```diff
@@ -0,0 +1,248 @@
+<?php
+/*
+ * Copyright 2005-2015 Centreon
+ * Centreon is developped by : Julien Mathis and Romain Le Merlus under
+ * GPL Licence 2.0.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation ; either version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, see <http://www.gnu.org/licenses>.
+ *
+ * Linking this program statically or dynamically with other modules is making a
+ * combined work based on this program. Thus, the terms and conditions of the GNU
+ * General Public License cover the whole combination.
+ *
+ * As a special exception, the copyright holders of this program give Centreon
+ * permission to link this program with independent modules to produce an executable,
+ * regardless of the license terms of these independent modules, and to copy and
+ * distribute the resulting executable under terms of Centreon choice, provided that
+ * Centreon also meet, for each linked independent module, the terms  and conditions
+ * of the license of that module. An independent module is a module which is not
+ * derived from this program. If you modify this program, you may extend this
+ * exception to your version of the program, but you are not obliged to do so. If you
+ * do not wish to do so, delete this exception statement from your version.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+if (!defined('SMARTY_DIR')) {
+    define('SMARTY_DIR', realpath('../vendor/smarty/smarty/libs/') . '/');
+}
+
+/*
+ * Bench
+ */
+function microtime_float()
+{
+    list($usec, $sec) = explode(" ", microtime());
+    return ((float)$usec + (float)$sec);
+}
+
+set_time_limit(60);
+$time_start = microtime_float();
+
+$advanced_search = 0;
+
+/*
+ * Include
+ */
+include_once(realpath(dirname(__FILE__) . "/../../../../bootstrap.php"));
+
+require_once "$classdir/centreonDB.class.php";
```

Comment:
! -> $classdir may not be defined
I found only one affectation in the "centreon.config.php.template" . But I may be wrong. Why don't you use a  $path = __DIR__ . "XXX" and use it instead ?